### DISCUSSION TITLE
not hiding ipv6 adress?

### DISCUSSION THREAD

**Discussion by UniHorseV3** [pivpn/pivpn#1435] [Support]
Followed this tutorial, and my ipv4 adress is my VPS's IPv4 adress but my IPv6 is still my old one?https://www.scaleway.com/en/docs/tutorials/pihole-vpn/

**Comment by orazioedoardo** [comment]
Open `/etc/openvpn/server.conf` and change [this line](https://github.com/pivpn/pivpn/blob/f80b0a7962d91862132c0a4abd65c1e67bd37bd7/files/etc/openvpn/server_config.txt#L18) to `push "redirect-gateway def1 ipv6"`. Save the file and restart openvpn with `systemctl restart openvpn`.

This change routes IPv6 through the VPN. It won’t actually work since the server is not configured to forward IPv6, but it should stop the leak, leaving you with just the IPv4 address.
